public/locales/auset-te.json

```diff
@@ -0,0 +1,426 @@
+      "setupTitle": "కోర్విడ్ కుట్రను సెటప్ చేయండి",
+      "setup": "<i>1.</i> మాతృభూమి క్లియరింగ్‌ను ఎంచుకోండి. మీకు నచ్చిన 1 <strong>యోధుడు</strong> మరియు 1 <strong>ప్లాట్ టోకెన్</strong>ని అక్కడ ఉంచండి.<br/><i>2.</i> 1 <strong>యోధుడిని</strong> ఉంచండి. > ప్రతి సూట్ క్లియరింగ్‌లో. <i>(మునుపటి దశను లెక్కించడం, మొత్తం 4 మంది యోధులను ఉంచండి.)</i>"
+    },
+    "cult": {
+      "name": "బల్లి కల్ట్(లిజార్డ్ కల్ట్)",
+      "summaryTitle": "లిజార్డ్ కల్ట్ గా ఆడటం",
+      "summary": "<p>లిజార్డ్ కల్ట్‌గా, మీరు ఇతర వర్గాలచే విస్మరించబడిన జీవులను అందిస్తారు. సరైన <strong>ఆచారాలు</strong> చేయడం ద్వారా మీరు విజయ పాయింట్లను స్కోర్ చేస్తారు, మీకు <strong>తోటలు</strong> ఉన్న చోట మీ చేతి మ్యాచింగ్ క్లియరింగ్‌ల నుండి కార్డ్‌లను బహిర్గతం చేయడం మరియు ఆచారాన్ని పూర్తి చేయడానికి కార్డ్‌లను విస్మరించడం. వెల్లడైన కార్డ్‌కి సరిపోలే క్లియరింగ్‌లలో మీకు ఎక్కువ గార్డెన్‌లు ఉంటే, మీరు ఎక్కువ పాయింట్లను స్కోర్ చేస్తారు.</p><p>మీ సౌమ్య వైఖరి అంటే మీరు మొదట మీ శత్రువులతో పోరాడలేరని అర్థం; అలా చేయడానికి, మీరు మీ అనుచరులను <strong>అకోలైట్‌లు</strong>గా మార్చాలి. ఇలా చేయడం ద్వారా, మీరు <strong>బహిష్కరించబడిన</strong> క్లియర్‌లలో <strong>కుట్రలు</strong> చేయవచ్చు, మునుపటి రౌండ్‌లో దావా ఎక్కువగా విస్మరించబడింది.</p><p>మీ <strong>ద్వేషం పక్షులు</strong>అంటే మీ ఆచారాలలో మీ పక్షి కార్డులు విపరీతంగా లేవు. మీ తోటలు మీ డ్రాగన్ ప్రభువు గురించిన వార్తలను ప్రచారం చేస్తాయి, <strong>యాత్రికుల</strong> సమూహాలను ఆకర్షిస్తాయి, కాబట్టి మీకు అక్కడ ఒక్క తోట అయినా ఉంటే మీరు క్లియరింగ్‌ను పరిపాలిస్తారు. చివరగా, మీ సహచరులు ఎల్లప్పుడూ దుష్టులపై <strong>ప్రతీకారం</strong> పొందుతారు: మీ యోధులలో ఒకరిని రక్షించేటప్పుడు తొలగించబడినప్పుడు, మీరు మరొక సహచరుడిని పొందుతారు.</p>",
+      "setupTitle": "లిజార్డ్ కల్ట్‌ని సెటప్ చేయండి",
+      "setup": "<i>1.</i> శత్రు భూభాగాలకు ఆనుకొని లేని స్వదేశీ క్లియరింగ్‌ను ఎంచుకోండి.<br/><i>2.</i> 4 <strong>యోధులు</strong> మరియు 1 సరిపోలే <strong> ఉంచండి తోట</strong>మీ స్వదేశంలో. 3 <strong>యోధులను</strong> దాని ప్రక్కనే ఉన్న క్లియరింగ్‌లలో వీలైనంత సమానంగా ఉంచండి.<br/><i>3.</i> మీ అకోలైట్స్ బాక్స్‌లో 2 <strong>యోధులను</strong> ఉంచండి.<br /><i>4.</i> మీ గార్డెన్స్ ట్రాక్‌లను <strong>గార్డెన్స్</strong>తో పూరించండి, మ్యాప్‌లో గార్డెన్‌కి ఎడమవైపున ఉన్న స్థలం మినహా.<br/><i>5.</i> ఉంచండి మీ <strong>అవుట్‌కాస్ట్ మార్కర్</strong> అవుట్‌కాస్ట్ బాక్స్‌లోని ఏదైనా స్థలంలో దాని అవుట్‌కాస్ట్ వైపు ఉంటుంది."
+    },
+    "duchy": {
+      "name": "భూగర్భ ప్రదేశం(అండర్గ్రౌండ్ డచి)",
```

Comment:
I think the phonetic translation is a nice touch, however it makes the faction name too long to fit neatly in the interface. I would suggest just using the faction name from the official Leder games translation, or if that doesn't exist use just the literal translation without the phonetic one.